Makefile

```diff
@@ -157,17 +157,22 @@ install:
 develop:
 	cd lib ; python setup.py develop
 
-.PHONY: wheel
-# Create a Python wheel file in dist/.
-wheel:
+.PHONY: distribution
+# Create Python distribution files in dist/.
+distribution:
 	# Get rid of the old build folder to make sure that we delete old js and css.
 	rm -rfv lib/build
-	cd lib ; python setup.py bdist_wheel --universal
-	# cd lib ; python setup.py bdist_wheel sdist
+	cd lib ; python setup.py bdist_wheel --universal sdist
+
+.PHONY: clean-package
+# Removes existing packages and creates distribution files in dist/.
+clean-package:
+	rm -rfv lib/dist
+	distribution
```

Comment:
Should this be "package" instead?

Reply:
good point. i was just getting tired in my testing and didn't want to wait for the slow parts of `make package` but usually we would want to do it all.